src/Algebra/FunctionProperties/LeftAction.agda

```diff
@@ -0,0 +1,53 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of functions, such as associativity and commutativity
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Relation.Binary
+
+-- The properties are parameterised by the two carriers and
+-- the result equality.
+
+module Algebra.FunctionProperties.LeftAction
```

Comment:
The `FunctionProperties` subfolder is being deprecated. We could try either `Algebra.Module.Definitions.LeftAction` or `Algebra.Definitions.LeftAction`?